[2021-07-02 05:43:43,387] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: test.print_date 2021-04-13T05:55:00+00:00 [queued]>
[2021-07-02 05:43:43,405] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: test.print_date 2021-04-13T05:55:00+00:00 [queued]>
[2021-07-02 05:43:43,405] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-07-02 05:43:43,405] {taskinstance.py:1068} INFO - Starting attempt 1 of 2
[2021-07-02 05:43:43,406] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-07-02 05:43:43,423] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): print_date> on 2021-04-13T05:55:00+00:00
[2021-07-02 05:43:43,428] {standard_task_runner.py:52} INFO - Started process 178429 to run task
[2021-07-02 05:43:43,434] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'test', 'print_date', '2021-04-13T05:55:00+00:00', '--job-id', '1323', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmphdh_u5o1', '--error-file', '/tmp/tmp4ktoik8o']
[2021-07-02 05:43:43,436] {standard_task_runner.py:77} INFO - Job 1323: Subtask print_date
[2021-07-02 05:43:43,495] {logging_mixin.py:104} INFO - Running <TaskInstance: test.print_date 2021-04-13T05:55:00+00:00 [running]> on host 9b3ed935c8bc
[2021-07-02 05:43:43,575] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@airflow.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=test
AIRFLOW_CTX_TASK_ID=print_date
AIRFLOW_CTX_EXECUTION_DATE=2021-04-13T05:55:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-04-13T05:55:00+00:00
[2021-07-02 05:43:43,576] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-07-02 05:43:43,576] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'date']
[2021-07-02 05:43:43,587] {subprocess.py:75} INFO - Output:
[2021-07-02 05:43:43,588] {subprocess.py:79} INFO - Fri Jul  2 05:43:43 UTC 2021
[2021-07-02 05:43:43,589] {subprocess.py:83} INFO - Command exited with return code 0
[2021-07-02 05:43:43,622] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=test, task_id=print_date, execution_date=20210413T055500, start_date=20210702T054343, end_date=20210702T054343
[2021-07-02 05:43:43,654] {taskinstance.py:1245} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2021-07-02 05:43:43,696] {local_task_job.py:151} INFO - Task exited with return code 0
